dsc/config.py

```diff
@@ -5,6 +5,12 @@
 
 import sentry_sdk
 
+WORKFLOWS = {
+    "test": {
+        "workflow-path": "tests.conftest.TestBaseWorkflow",
```

Comment:
If you are going to go the route of a dictionary/JSON object to define what workflows are called, and where to find them, I might suggest changing `workflow-path` to something like `workflow_classpath` which I think explicitly indicates this is the import pattern for the workflow class itself.